README.md

```diff
@@ -68,6 +68,38 @@ meaning you will be able to access the items using `Items.A` and `Items.B`.
 
 ---
 
+The above works recursively with nested directories as well:
+
+```
+|- index.js
+|- dir
+    |- a.js
+    |- c.js
+    |- nested
+        |- b.js
+```
+
+the following JS:
+
+```javascript
+import * as Items from './dir';
+```
+
+will be compiled to:
+
+```javascript
+const Items = {};
+import _wcImport from "./dir/a";
+Items.A = _wcImport;
+import _wcImport1 from "./dir/c";
+Items.C = _wcImport1;
+const Items.Nested = {};
```

Comment:
Shouldn't this just be `Items.Nested = {}`?

Reply:
I believe so actually.